src/services/lnd/utils.ts

```diff
@@ -1,39 +1,36 @@
+import {
```

Comment:
it feels my vs code wants to change import order. is there a way a setting to fix this @samerbuna ?

Reply:
It would be somewhere in your saved settings - cmd+shift+p, open settings(json)

Inspect that for any sort/organize settings for imports